### PR TITLE
Sort events by starting time. Fixes #172

### LINKED ISSUE #172: Events should be organized by time in the Calendar
**Describe the bug**
Events appear to be sorted by the order in which they are entered and not by the time of the event
Screen Shot 2022-12-14 at 11 10 22 AM

**To Reproduce**
1. Go to ['Together'](https://together.cyclic.app/) or start the server in your development enviroment
2. Login with discord
3. Click on Navigate to Calendar Page

**Expected behavior**
Events should be sorted by earliest to latest event in a given day.

**Screenshots**
Included in body

*PR should go into development branch*

**Desktop (please complete the following information):**
Appears on all devices

### PR DESCRIPTION
**Pull request by romanstetsyk**
# Description

Sort events by starting time

## Type of change

Please select everything applicable. Please, do not delete any lines.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Issue
- [x] Is this related to a specific issue? Is so please specify: #172 

# Checklist:

- [x] I have executed `npm run lint` and resolved any outstanding errors. Most issues can be solved by executing `npm run format`
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Caleb-Cohen** [comment]
Just curious. Why the spread operator? Would this work without it?

**Comment by romanstetsyk** [comment]
From what I read, it's a bad practice to mutate state or props directly. Although it works without spread

File: client/src/features/calendar/DayCard.js [hunk 1]
```diff
@@ -4,15 +4,20 @@ import Event from "./Event";
 const DayCard = ({ day, month, events }) => {
   // Styling for bullet point
 
+  // Sort events by startAt property
+  let sortedEvents = [...events].sort(
```
Comment (timmyichen) [inline client/src/features/calendar/DayCard.js:8]: could this just be `events.sort`?

**Comment by MatthewBozin** [comment]
@Caleb-Cohen there's no downside to using the spread operator here instead of .split(), just personal preference.

@romanstetsyk agreed, it's a good call to copy instead of mutate here.

**Review by MatthewBozin — APPROVED** [review]
This looks good, worked when I tested it. Well done!

**Pull request merged**